src/pages/organizations/pages/organizations-page/index.tsx

```diff
@@ -77,20 +77,15 @@ const OrganizationsPage: FC<Props> = ({
       <div className='row'>
         {filterOrganizationsByName(searchQuery).map(
           (
-            {
-              id,
-              organization: { name },
-              dataset_count,
-              dataservice_count
-            }: Organization,
+            { id, name, prefLabel, datasetCount, dataserviceCount }: OrgSummary,
```

Comment:
Usikker, men typen bør være overflødig her

```suggestion
            { id, name, prefLabel, datasetCount, dataserviceCount },
```

Reply:
b25a394